packages/ddp-server/livedata_server.js

```diff
@@ -2,6 +2,33 @@ DDPServer = {};
 
 var Fiber = Npm.require('fibers');
 
+// Publication strategies define how we threat data from published cursors on collection-level
+// This allows someone to:
+// - Make a trade-off between bandwidth and memory use
+// - Have special (non-mongo) collections like volatile message queues
+const publicationStrategies = {
+  // The 'no merge no history' strategy means we send all publication data 
+  // directly to the client. It does not remember what it has send
+  // So it will also not trigger removed messages when a subscription stops
+  // This should only be chosen for special use cases like send-and-forget queues
+  NO_MERGE_NO_HISTORY: 'no_merge_no_history',
+  // No merge is similar to 'no merge no history'
+  // But it will remember the id's it has send to the client
+  // so it can remove them when a subscription stops
+  // It can be used when a collection is only used in a single publication
+  NO_MERGE: 'no_merge',
+  // XXX: We could implement client side merge
+  // By moving the session collection view to the client
+  // And sending subscription id's along the added/changed/removed messages
+  // It can be used when we want to save server memory but still have the same
+  // semantics over multiple publications of one collection
+  // CLIENT_MERGE: 'client_merge',
```

Comment:
Do we want to include this in this PR? Otherwise I would remove from the code and just open an issue for this use-case.

Reply:
@filipenevola I have removed this.